packages/next-codemod/transforms/__tests__/new-link.test.js

```diff
@@ -1,23 +1,16 @@
 /* global jest */
 jest.autoMockOff()
 const defineTest = require('jscodeshift/dist/testUtils').defineTest
+const { readdirSync } = require('fs')
+const { join } = require('path')
 
-const fixtures = [
-  'link-a',
-  'move-props',
-  'add-legacy-behavior',
-  'excludes-links-with-legacybehavior-prop',
-  'children-interpolation',
-  'spread-props',
-  'link-string',
-  'styled-jsx',
-]
+const fixtureDir = 'new-link'
```

Comment:
I wrote this code for the other codmod and reused it here.

It allows you do have a single `fixtureDir` variable and then figures out the rest.

I could move it into a shared util if we think its useful.